core/templates/base-components/oppia-footer.component.html

```diff
@@ -1,6 +1,37 @@
 <footer class="oppia-footer" role="contentinfo">
+  <div class="oppia-footer-newsletter-container">
+    <div class="oppia-footer-newsletter">
+      <div class="oppia-newsletter-content">
+        <h3 [innerHTML]="'I18N_FOOTER_NEWSLETTER_HEADING' | translate"></h3>
+        <p [innerHTML]="'I18N_FOOTER_NEWSLETTER_TEXT' | translate"></p>
+      </div>
+      <div class="oppia-footer-newsletter-input">
+        <label id="email"
+               [innerHTML]="'I18N_FOOTER_NEWSLETTER_EMAIL' | translate"
+        >
+        </label>
+        <div class="oppia-newsletter-input">
+          <input type="email"
+                 [(ngModel)]="emailAddress"
+                 [ngModelOptions]="{standalone: true}"
+                 required="true"
+                 aria-labelledby="email">
+          <button class="oppia-footer-newsletter-button"
+                  [disabled]="!validateEmailAddress()"
+                  (click)="subscribeToMailingList()">
+            <span [innerHTML]="'I18N_FOOTER_NEWSLETTER_SUBSCRIBE' | translate"></span>
+          </button>
+        </div>
+      </div>
+    </div>
+  </div>
   <div class="oppia-footer-container">
     <div class="row row-cols-2">
+      <div class="oppia-mission-volunteer col-5">
+        <p class="oppia-mission" [innerHTML]="'I18N_FOOTER_OPPIA_MISSION' | translate"></p>
+        <p class="oppia-donate-volunteer" [innerHTML]="'I18N_FOOTER_OPPIA_MISSION_DONATE_VOLUNTEER' | translate"></p>
+      </div>
+      <hr class="oppia-footer-separator">
       <div class="col">
         <p [innerHTML]="'I18N_FOOTER_ABOUT_ALL_CAPS' | translate"></p>
```

Comment:
Ditto

Reply:
Hey, When using interpolation in `oppia-donate-volunteer`, the tags are being interpreted as is. As a result, I had to stick with it. Thanks

Reply:
Does this not work? `<p class="oppia-donate-volunteer">{ 'I18N_FOOTER_OPPIA_MISSION_DONATE_VOLUNTEER' | translate }</p>`

Reply:
I am sorry if I couldn't explain it properly to you :(. I tried doing what you suggested earlier. but on doing so. the tags in the `i18n` values were rendered as is.

![Screenshot 2023-08-06 090425](https://github.com/oppia/oppia/assets/76906722/1170b322-5b96-422d-b45f-00c152ad685d)

But with the `[innerHtml]` approach they seem to render as HTML. Correct me if I am wrong, Thanks

Reply:
Umm... Weird can you share the piece of code that causes this issue with internpolation? (also looks like I made a mistake in the comment, use a double {{ }} to enclose the data to be interpolated)

Reply:
```html
      <div class="oppia-mission-volunteer col-7">
        <p class="oppia-mission">{{ 'I18N_FOOTER_OPPIA_MISSION' | translate }}</p>
        <p class="oppia-donate-volunteer">{{ 'I18N_FOOTER_OPPIA_MISSION_DONATE_VOLUNTEER' | translate }}</p>
      </div>
```

In this code, regardless of whether we do it, the i18n will be rendered as plain text. So, in this scenario, any tags in the i18n

```json
"I18N_FOOTER_OPPIA_MISSION_DONATE_VOLUNTEER": "<a href=\"/donate\">Donate</a> or <a href=\"/volunteer\">Volunteer</a> today!",
```

They will, however, be shown as HTML if you use `[innerHtml]`. So, I believe we should use `[innerHtml]` in this situation. What do you think?

Reply:
Ah, I see, yes that is correct, didn't know that the variable pointed to HTML code.